contents/handbook/engineering/support-hero.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Support Hero
+sidebar: Handbook
+showTitle: true
+---
+
+Every release cycle, we assign one person to be the "Support Hero." Their primary job is to help customers, debug/fix issues, and debug/fix sentry errors, in that order of priority. All other work takes a back seat while they doing support.
+
+There are a couple of channels that customer requests come in so make sure you keep an eye on all of them:
```

Comment:
It would be good if this could be harmonised with or linked to the existing [Customer Success page](https://posthog.com/handbook/growth/customer-success)

Reply:
ah, i guess we didn't really need this page, but customer success is something different in my mind to customer support, even though the contents of that page are quite similar. Worth porting over that content here instead?

Reply:
To be honest that page in 'Growth' should really be renamed 'Customer Support' as it's not really Customer Success anyway. I don't really mind where it sits in either case though! Probably Engineering makes more sense given our current system of everyone on support being engineers?

Reply:
Yeah agree.